Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -78,6 +81,14 @@ public extension WKUserContentController {
           }
         }
         """
+
+        let userScriptsWithoutDatadog = userScripts.filter { script in
+            return !script.source.starts(with: jsPrefix)
+        }
+        if userScriptsWithoutDatadog.count != userScripts.count {
+            removeAllUserScripts()
+            userScriptsWithoutDatadog.forEach { addUserScript($0) }
+        }
```

Comment:
The problem we're solving is that calling `trackDatadogEvents(in:)` more than once results with a crash:
```swift
controller.trackDatadogEvents(in hosts: ["foo.com"])
controller.trackDatadogEvents(in hosts: ["foo.com"]) // 🔥 crash
```

But this solution looks very odd and unobvious: each time the API is called we're removing **all user scripts** and then re-adding all but Datadog one.

Can't we just early return in this API if it was already called?
```swift
func trackDatadogEvents(in hosts: Set<String>) {
   guard !isAlreadyTracked(self) else {
      userLogger.warn("`trackDatadogEvents(in:)` was called more than once for the same WebView. Second call will be ignored. Make sure you call it only once.")
      return
   }

   // ...
}
```